spec/cypress/support/navigation.js

```diff
@@ -0,0 +1,131 @@
+// Checking the baseurl -- mco or avalon
+function getPlatform() {
+  const baseUrl = Cypress.config('baseUrl') || '';
+  if (baseUrl.includes('avalon') || baseUrl.includes('localhost'))
+    return 'avalon';
+  if (baseUrl.includes('mco')) return 'mco';
```

Comment:
I'd like to keep MCO-specific logic out of the core avalon codebase.  Would it be possible to convert these functions into cypress environment variables?